src/index.ts

```diff
@@ -1,12 +1,13 @@
-import express from 'express'
+require('dotenv').config();
+import express from 'express';
 
-const app = express()
-const port = 3000
+const app = express();
+const port = process.env.PORT;
```

Comment:
I propose to add a default PORT + `process.env.PORT` is not set in `package.json`

Reply:
Added a default port 👍 

Yes it's not in the `package.json`, why do we need it there?

Reply:
because, as it is not set in the code, it should be at least set in scripts in package.json

Reply:
It will be set either in our deployment environment setup, or locally in the `.env` file, that's why I put `require('dotenv').config();` in the begining - to bring variables from `.env` into the process. For local development you may copy `.env-sample` and rename it to `.env`, replace sensitive variables and everything will work then.
More on how it works with `dotenv`: https://www.npmjs.com/package/dotenv